test/controllers/healthcare/HealthcareList1ControllerSpec.scala

```diff
@@ -33,12 +36,17 @@ import play.api.test.Helpers._
 import repositories.SessionRepository
 import views.html.healthcare.HealthcareList1View
 
-class HealthcareList1ControllerSpec extends SpecBase with ScalaFutures with OptionValues with IntegrationPatience {
+import scala.concurrent.Future
+
+class HealthcareList1ControllerSpec extends SpecBase with ScalaFutures with OptionValues with IntegrationPatience with MockitoSugar with BeforeAndAfterEach {
```

Comment:
added beforeAndAfterEach but doesn't use it

Reply:
Removed from all unused tests